src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -111,10 +111,10 @@ impl TargetInfo {
             .args(&rustflags)
             .env_remove("RUSTC_LOG");
 
-        let mut bitcode_in_rlib_test = process.clone();
-        bitcode_in_rlib_test.arg("-Cbitcode-in-rlib");
-        let supports_bitcode_in_rlib = match kind {
-            CompileKind::Host => Some(rustc.cached_output(&bitcode_in_rlib_test).is_ok()),
+        let mut embed_bitcode_test = process.clone();
+        embed_bitcode_test.arg("-Cbitcode-in-rlib");
```

Comment:
Note that the naming here is intentionally a bit odd. This is being renamed in https://github.com/rust-lang/rust/pull/71716 (and will get backported to beta). The naming here is intended to represent what should be the final state of things while also working with today's nightly.